_Group Projects/Group_2/Week_5/program.do

```diff
@@ -0,0 +1,60 @@
+/* Group 2
+Week 5 Assignment
+Authors: Felipe, Yuri, Moataz, and Yash 
+Last Modified: October 9, 2022 (09/10/2022)
+Program .do file */
+
+cd "/Users/devakid/Desktop/Git/econ490-fall22/_Group Projects/Group_2/Week_5"
+cap prog drop se_and_outlier
+prog define se_and_outlier 
+syntax anything
+
+preserve
+
+/* Identifying and dropping string variables before generating a covariance 
+matrix and mapping the distribution of standard errors
+*/
+ds, has(type string)
+drop `r(varlist)'
+
+// Generating a local for our dependent variable 
+local dep_var: word 1 of `anything'
+
+// Creating a list of the remaining non-string variables 
+ds `dep_var', not
+local othervar `r(varlist)'
+
+// Creating a loop to generate a covariance matrix for assessing collinearity issues
+foreach var in `othervar' {
+	corr, cov
+}
+
+/* Creating a loop to construct plots which check for the nature of the 
+distribution of the error terms
+*/ 	
+foreach var in `othervar' {
```

Comment:
Your program outputs many tables, so I think it would be helpful if you think more deeply about what information would be most useful to the user. Is it necessary to output the full regression table for every specification? If you think not, you can regress quietly and display different key macros from the regressions. If you think the full regression table is necessary for each specification, you can add text break up the output and orient the user. For example, you could add 
display "Regression of `dep_var' on `var'"
or another descriptive title to the loop.